recipes/root/all/conanfile.py

```diff
@@ -0,0 +1,300 @@
+        tools.replace_in_file(
+            os.sep.join((self._rootsrcdir, "CMakeLists.txt")),
+            "project(ROOT)",
+            """project(ROOT)
+            find_package(OpenSSL REQUIRED)
+            set(OPENSSL_VERSION ${OpenSSL_VERSION})
+            find_package(LibXml2 REQUIRED)
+            set(LIBXML2_INCLUDE_DIR ${LibXml2_INCLUDE_DIR})
+            set(LIBXML2_LIBRARIES ${LibXml2_LIBRARIES})
+            find_package(SQLite3 REQUIRED)
+            set(SQLITE_INCLUDE_DIR ${SQLITE3_INCLUDE_DIRS})
+            set(SQLITE_LIBRARIES SQLite::SQLite)
+            """,
+        )
+
+    def _fix_source_permissions(self):
+        # Fix execute permissions on scripts
+        scripts = [
+            filename
+            for pattern in (
+                "**" + os.sep + "configure",
+                "**" + os.sep + "*.sh",
+                "**" + os.sep + "*.csh",
+                "**" + os.sep + "*.bat",
+            )
+            for filename in glob(pattern, recursive=True)
+        ]
+        for s in scripts:
+            self._make_file_executable(s)
+
+    def _make_file_executable(self, filename):
+        st = os.stat(filename)
+        os.chmod(filename, st.st_mode | stat.S_IEXEC)
+
+    @property
+    def _configured_cmake(self):
+        if self._cmake is None:
+            self._move_findcmake_conan_to_root_dir()
+            self._cmake = CMake(self)
+            version = self.version.replace("v", "")
+            cmakelibpath = ";".join(self.deps_cpp_info.lib_paths)
+            cmakeincludepath = ";".join(self.deps_cpp_info.include_paths)
+            self._cmake.configure(
+                source_folder="root-{}".format(version),
+                defs={
+                    # TODO: Remove BUILD_SHARED_LIBS option when hooks issue is resolved
+                    # (see: https://github.com/conan-io/hooks/issues/252)
+                    "BUILD_SHARED_LIBS": "ON",
+                    "fail-on-missing": "ON",
+                    "CMAKE_CXX_STANDARD": self._cmake_cxx_standard,
+                    "gnuinstall": "OFF",
+                    "soversion": "ON",
+                    # Disable builtins and use Conan deps where available
+                    "builtin_pcre": "OFF",
+                    "builtin_lzma": "OFF",
+                    "builtin_zstd": "OFF",
+                    "builtin_lz4": "OFF",
+                    "builtin_glew": "OFF",
+                    "builtin_openssl": "OFF",
+                    "builtin_fftw3": "OFF",
+                    "builtin_cfitsio": "OFF",
+                    "builtin_davix": "OFF",
+                    "builtin_tbb": "OFF",
+                    # Enable builtins where there is no Conan package
+                    "builtin_xxhash": "ON",
+                    "builtin_afterimage": "ON",
+                    "builtin_gsl": "ON",
+                    "builtin_gl2ps": "ON",
+                    "builtin_ftgl": "ON",
+                    "builtin_vdt": "ON",
```

Comment:
Can you add some `FIXME`s about these missing recipes?

Reply:
Added FIXME comments in 1dcf0be6.